nemo/export/trt_llm/nemo_utils.py

```diff
@@ -213,7 +215,7 @@ def to_word_list_format(word_dict: List[List[str]], tokenizer=None):
     # We use a similar trick as in NeMo to deal with the fact that the encoding of a single word
     # can't always be trusted. See
     #   https://github.com/NVIDIA/NeMo/blob/bb575b72fd0be51ae10cc77d9f89ddb9e9d3b96d/nemo/collections/nlp/modules/common/text_generation_strategy.py#L229
```

Comment:
Can you please edit this comment? It looks like this was copy/pasted from NeMo-FW and it seems weird now that it's in NeMo. Something like:
```
# We use a similar trick as when generating text to deal with the fact that the encoding of a single word
# can't always be trusted. See
#   https://github.com/NVIDIA/NeMo/blob/bb575b72fd0be51ae10cc77d9f89ddb9e9d3b96d/nemo/collections/nlp/modules/common/text_generation_strategy.py#L229
```

Ideally this logic would be factored in a single place but I'm not sure if this is easy to do (may also not be worth the trouble since I believe the generation code will be deprecated at some point in favor of its upcoming mcore version).

Reply:
Sure will do. We did ask TRTLLM for string level matching but they felt like it was infeasible to implement.

Reply:
Yeah there's no obvious way to achieve this (that I know of). The best I can think of would be to have an option to regularly check the last chunk of text that has been generated to see if/when we need to stop (doing it on each step and over the whole input as in NeMo is overkill and very inefficient).